module-infrastructure/persistence-db/src/main/kotlin/site/yourevents/invitationthumnail/repository/InvitationThumbnailRepository.kt

```diff
@@ -0,0 +1,16 @@
+package site.yourevents.invitationthumnail.repository
+
+import org.springframework.stereotype.Repository
+import site.yourevents.invitationthumnail.domain.InvitationThumbnail
+import site.yourevents.invitationthumnail.entity.InvitationThumbnailEntity
+import site.yourevents.invitationthumnail.port.out.InvitationThumbnailPersistencePort
+
+@Repository
+class InvitationThumbnailRepository(
+    private val invitationThumbnailJPARepository: InvitationThumbnailJPARepository
+) : InvitationThumbnailPersistencePort {
+    override fun saveInvitationThumbnail(invitationThumbnail: InvitationThumbnail): InvitationThumbnail {
+        return invitationThumbnailJPARepository.save(InvitationThumbnailEntity.from(invitationThumbnail))
+            .toDomain()
```

Comment:
마찬가지로 적절한 개행을 통해 가독성을 높일 수 있을 것 같습니다..!
```suggestion
        return invitationThumbnailJPARepository.save(
        InvitationThumbnailEntity.from(invitationThumbnail))
            .toDomain()
```

Reply:
개행 진행하였습니다!!